sklearn/metrics/ranking.py

```diff
@@ -109,10 +105,14 @@ def auc(x, y, reorder=False):
 
 
 def average_precision_score(y_true, y_score, average="macro",
-                            sample_weight=None):
+                            sample_weight=None, interpolation=None):
     """Compute average precision (AP) from prediction scores
 
-    This score corresponds to the area under the precision-recall curve.
+    Optionally, this will compute an eleven-point interpolated average
+    precision score: for each of the 11 evenly-spaced target recall values
```

Comment:
maybe say eleven in words to make it easier to spot the connection to the interpolation arg below